### DISCUSSION TITLE
Improvement: Add `CreateLocalAttribute` to combine `CreateAttribute` and `RunLocalAttribute` in one.

### DISCUSSION THREAD

**Discussion by ossendorf-at-hoelscher** [MarimerLLC/csla#3744] [Ideas]
Idea:
It's not necessary since it's possible with the combination but I think it would be a nice usability addition when you can use a new attribute called `CreateLocalAttribte`. It behaves exactly like the other two combined on one method.

**Comment by rockfordlhotka** [comment]
Or maybe `[Create(RunLocal = true)]`?

**Reply by StefanOssendorf** [reply]
That sounds even better!
